pom.xml

```diff
@@ -647,12 +649,13 @@
                 <version>${mokito-core.version}</version>
                 <scope>test</scope>
             </dependency>
-<dependency>
-  <groupId>com.zebrunner</groupId>
-  <artifactId>agent-testng</artifactId>
-  <version>RELEASE</version>
-  <scope>test</scope>
-</dependency>
+
+            <dependency>
+              <groupId>com.zebrunner</groupId>
+              <artifactId>agent-testng</artifactId>
+              <version>RELEASE</version>
```

Comment:
reuse `zebrunner-agent.version` maven properties you declared at the top

Reply:
done